fluentd/configs.d/openshift/filter-k8s-record-transform.conf

```diff
@@ -1,30 +1,35 @@
 <filter kubernetes.journal.container**>
-  type record_transformer
+  @type record_transformer
   enable_ruby
   <record>
-    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # FixMe
+    # Top level hostname field; it comes from record['HOSTNAME']
+    # The index mapping has a field record['kubernetes']['hostname'];
+    # it comes from record['kubernetes']['host'] or File.open('/etc/docker-hostname')
+    # Needs to set a field inside a hash.
+    # Plus we'd better avoid the file IO.
+    hostname ${(record['kubernetes']['host'] rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
     # log is set if MESSAGE is JSON valued and has err or msg field
     message ${record['MESSAGE'] || log}
-    version 1.3.0
+    level ${record['PRIORITY']}
+    ipaddr4 "#{ENV['IPADDR4']}"
+    ipaddr6 "#{ENV['IPADDR6']}"
+    version 1.4.0
     time ${Time.at((record["_SOURCE_REALTIME_TIMESTAMP"] || record["__REALTIME_TIMESTAMP"]).to_f / 1000000.0).to_datetime.rfc3339(6)}
```

Comment:
my concern is that I see that in the ElasticSearch document... not sure that it can be fixed, but it would be nice to get rid of that field and keep it only in `@timestamp`